inc/html.class.php

```diff
@@ -4708,11 +4685,11 @@ static function parseAttributes($options = []) {
     *
     * @since 0.85
     *
-    * @param $key       The name of the attribute to create
-    * @param $value     The value of the attribute to create.
+    * @param string $key       name of the attribute to create
+    * @param mixed  $value     value of the attribute to create.
```

Comment:
`string|array`

Reply:
which argument do you mean? if you refer about the value the mixed argument could be a valid option